microsoft-os-tracking.zeek

```diff
@@ -0,0 +1,36 @@
+@load policy/frameworks/software/windows-version-detection.zeek
+
+module OSPTracking;
+
+export {
+}
+
+event HTTP::log_http(rec: HTTP::Info) &priority=5
+	{
+	if ( rec?$host && rec?$user_agent
+	    && /crl\.microsoft\.com/ in rec$host
+	    && /Microsoft-CryptoAPI\// in rec$user_agent )
+		{
+		if ( rec$user_agent !in OS::crypto_api_mapping )
```

Comment:
Does this still work reliably? The crypto-apu-mapping table seems super outdated.